### DISCUSSION TITLE
Cycle Through Wallpapers (Slideshow Background Feature)

### DISCUSSION THREAD

**Discussion by AkiraKusanagi** [EmiyaSyahriel/CrossLauncher#62] [Ideas]
Would you be able to implement this? The PS3 had this for certain dynamic themes (Or just random wallpapers....) where it cycled through backgrounds.

**Comment by EmiyaSyahriel** [comment]
Aren't there some live wallpaper that implements this feature in Play Store?

e.g : [Slideshow Wallpaper](https://play.google.com/store/apps/details?id=io.github.doubi88.slideshowwallpaper) (open-source, github project page [here](https://github.com/Doubi88/SlideshowWallpaper))
